codebuild/bin/install_libFuzzer.sh

```diff
@@ -44,7 +44,7 @@ mkdir -p "$LIBFUZZER_INSTALL_DIR"/lib && cp libFuzzer.a "$LIBFUZZER_INSTALL_DIR"
 
 # Run AFL instead of libfuzzer if AFL_FUZZ is set. Not compatible with fuzz coverage.
 if [[ "$AFL_FUZZ" == "true" && "$FUZZ_COVERAGE" != "true" ]]; then
-	mkdir -p "$LIBFUZZER_INSTALL_DIR" && curl https://raw.githubusercontent.com/google/clusterfuzz/master/docs/setting-up-fuzzing/build_afl.bash > "$LIBFUZZER_INSTALL_DIR"/build_afl.bash
+	mkdir -p "$LIBFUZZER_INSTALL_DIR" && curl https://raw.githubusercontent.com/google/clusterfuzz/ac5ac9e4604ea03cfd643185ad1e3800e952ea44/docs/setting-up-fuzzing/build_afl.bash > "$LIBFUZZER_INSTALL_DIR"/build_afl.bash
```

Comment:
Can you put a TODO comment here with a link to an issue?

Reply:
> Do we have a backlog issue to make our build work with the latest code?

I created one.

Reply:
In that case, I would put a comment to briefly describe the issue and why we are pinning to this commit.

Reply:
Maybe say: 

`Clusterfuzz's bash script changed from AFL to AFL++ on [insert date], this commit [insert commit id] pins the script to version [insert version number] until we support AFL++.`